sample-apps/00-fluentBit/kubernetes/fluentbit.yaml

```diff
@@ -147,7 +159,7 @@ spec:
     spec:
       containers:
         - name: fluent-bit
-          image: fluent/fluent-bit:latest
+          image: fluent/fluent-bit:2.2.2
```

Comment:
Is this the last version before Parse starts breaking?

Reply:
Yes, I tested back those versions, find out that fluent-bit version 2.2.2 is working. All version after that from 3.0.0 is having breaking change.